app/backend/graphql/task/mutations/createTaskMutationField.ts

```diff
@@ -0,0 +1,29 @@
+import { arg, intArg, mutationField, nullable, stringArg } from 'nexus'
+import { DateScalar } from '../../scalars/date'
+import { Task } from '../task'
+
+export const createTaskMutationField = mutationField('createTaskMutation', {
+  type: Task,
+  description: 'Create a new Task',
+  args: {
+    title: stringArg({ description: 'Title for the task' }),
+    projectId: intArg(),
+    startDate: nullable(arg({ type: DateScalar })),
+    endDate: nullable(arg({ type: DateScalar })),
+  },
+  authorize: (_source, _arguments, context) => !!context.session?.user.id,
```

Comment:
Also add the check, that the user is member of the project.

Reply:
done